### ISSUE TITLE
Concurrency Setting Not Persisting - Bug #3150 - Fix appears to have been regressed 

### ISSUE THREAD

**Issue by axiomitdavid** [Azure/LogicAppsUX#3199]
### Describe the Bug with repro steps

Set Concurrency Control to ON - Change degree of parallelism if desired
Save Workflow
Re-Open Workflow in Designer - Concurrency Control is off

### What type of Logic App Is this happening in?

Standard (VSCode)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Workflow JSON

```json
{
    "definition": {
        "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
        "actions": {
            "Get_GroupPolicyConfigurations": {
                "type": "Http",
                "inputs": {
                    "uri": "https://graph.microsoft.com/beta/deviceManagement/groupPolicyConfigurations",
                    "method": "GET",
                    "authentication": {
                        "type": "ActiveDirectoryOAuth",
                        "tenant": "@{triggerBody()?['tenantId']}",
                        "audience": "https://graph.microsoft.com",
                        "clientId": "@{parameters('axiomitcustomerappappid')}",
                        "secret": "@{parameters('axiomitcustomerappsecret')}"
                    }
                },
                "runAfter": {},
                "runtimeConfiguration": {
                    "contentTransfer": {
                        "transferMode": "Chunked"
                    }
                }
            },
            "Parse_Group_Policy_Configurations": {
                "type": "ParseJson",
                "inputs": {
                    "content": "@body('Get_GroupPolicyConfigurations')",
                    "schema": {
                        "type": "object",
                        "properties": {
                            "@@odata.context": {
                                "type": "string"
                            },
                            "value": {
                                "type": "array",
                                "items": {
                                    "type": "object",
                                    "properties": {
                                        "createdDateTime": {
                                            "type": "string"
                                        },
                                        "displayName": {
                                            "type": "string"
                                        },
                                        "description": {
                                            "type": "string"
                                        },
                                        "roleScopeTagIds": {
                                            "type": "array"
                                        },
                                        "policyConfigurationIngestionType": {
                                            "type": "string"
                                        },
                                        "id": {
                                            "type": "string"
                                        },
                                        "lastModifiedDateTime": {
                                            "type": "string"
                                        }
                                    },
                                    "required": [
                                        "createdDateTime",
                                        "displayName",
                                        "description",
                                        "roleScopeTagIds",
                                        "policyConfigurationIngestionType",
                                        "id",
                                        "lastModifiedDateTime"
                                    ]
                                }
                            }
                        }
                    }
                },
                "runAfter": {
                    "Get_GroupPolicyConfigurations": [
                        "SUCCEEDED"
                    ]
                }
            },
            "For_each_Group_Policy_Configuration": {
                "type": "Foreach",
                "foreach": "@body('Parse_Group_Policy_Configurations')?['value']",
                "actions": {
                    "Sub-Folder_Name": {
                        "type": "Compose",
                        "inputs": "@replace(replace(replace(replace(replace(replace(replace(replace(replace(item()?['displayName'], '<', '_'), '>', '_'), ':', '_'), '\"', '_'), '/', '_'), '\\\\', '_'), '|', '_'), '?', '_'), '*', '_')"
                    },
                    "Get_GroupPolicyConfiguration_Definition_Values": {
                        "type": "Http",
                        "inputs": {
                            "uri": "https://graph.microsoft.com/beta/deviceManagement/groupPolicyConfigurations/@{items('For_each_Group_Policy_Configuration')?['id']}/definitionValues",
                            "method": "GET",
                            "authentication": {
                                "type": "ActiveDirectoryOAuth",
                                "tenant": "@{triggerBody()?['tenantId']}",
                                "audience": "https://graph.microsoft.com",
                                "clientId": "@{parameters('axiomitcustomerappappid')}",
                                "secret": "@{parameters('axiomitcustomerappsecret')}"
                            }
                        },
                        "runAfter": {
                            "Sub-Folder_Name": [
                                "SUCCEEDED"
                            ]
                        },
                        "runtimeConfiguration": {
                            "contentTransfer": {
                                "transferMode": "Chunked"
                            }
                        }
                    },
                    "Parse_Group_Policy_Configurations_Definition_Values": {
                        "type": "ParseJson",
                        "inputs": {
                            "content": "@body('Get_GroupPolicyConfiguration_Definition_Values')",
                            "schema": {
                                "type": "object",
                                "properties": {
                                    "@@odata.context": {
                                        "type": "string"
                                    },
                                    "value": {
                                        "type": "array",
                                        "items": {
                                            "type": "object",
                                            "properties": {
                                                "createdDateTime": {
                                                    "type": "string"
                                                },
                                                "enabled": {
                                                    "type": "boolean"
                                                },
                                                "configurationType": {
                                                    "type": "string"
                                                },
                                                "id": {
                                                    "type": "string"
                                                },
                                                "lastModifiedDateTime": {
                                                    "type": "string"
                                                }
                                            },
                                            "required": [
                                                "createdDateTime",
                                                "enabled",
                                                "configurationType",
                                                "id",
                                                "lastModifiedDateTime"
                                            ]
                                        }
                                    }
                                }
                            }
                        },
                        "runAfter": {
                            "Get_GroupPolicyConfiguration_Definition_Values": [
                                "SUCCEEDED"
                            ]
                        }
                    },
                    "For_each_GroupPolicyConfigurationDefinitionValue": {
                        "type": "Foreach",
                        "foreach": "@body('Parse_Group_Policy_Configurations')?['value']",
                        "actions": {
                            "Parse_GroupPolicyDefintionPresentations": {
                                "type": "ParseJson",
                                "inputs": {
                                    "content": "@body('Get_GroupPolicyConfiguration_Definition_Values')",
                                    "schema": {
                                        "type": "object",
                                        "properties": {
                                            "@@odata.context": {
                                                "type": "string"
                                            },
                                            "value": {
                                                "type": "array",
                                                "items": {
                                                    "type": "object",
                                                    "properties": {
                                                        "@@odata.type": {
                                                            "type": "string"
                                                        },
                                                        "lastModifiedDateTime": {
                                                            "type": "string"
                                                        },
                                                        "createdDateTime": {
                                                            "type": "string"
                                                        },
                                                        "id": {
                                                            "type": "string"
                                                        },
                                                        "value": {
                                                            "type": "string"
                                                        }
                                                    },
                                                    "required": [
                                                        "@@odata.type",
                                                        "lastModifiedDateTime",
                                                        "createdDateTime",
                                                        "id",
                                                        "value"
                                                    ]
                                                }
                                            }
                                        }
                                    }
                                },
                                "runAfter": {
                                    "Get_GroupPolicyDefinitionPresentations": [
                                        "SUCCEEDED"
                                    ]
                                }
                            },
                            "Get_GroupPolicyDefinitionPresentations": {
                                "type": "Http",
                                "inputs": {
                                    "uri": "https://graph.microsoft.com/beta/deviceManagement/groupPolicyConfigurations/@{items('For_each_Group_Policy_Configuration')?['id']}/definitionValues/@{items('For_each_GroupPolicyConfigurationDefinitionValue')?['id']}",
                                    "method": "GET",
                                    "authentication": {
                                        "type": "ActiveDirectoryOAuth",
                                        "tenant": "@{triggerBody()?['tenantId']}",
                                        "audience": "https://graph.microsoft.com",
                                        "clientId": "@{parameters('axiomitcustomerappappid')}",
                                        "secret": "@{parameters('axiomitcustomerappsecret')}"
                                    }
                                },
                                "runtimeConfiguration": {
                                    "contentTransfer": {
                                        "transferMode": "Chunked"
                                    }
                                }
                            }
                        },
                        "runAfter": {
                            "Parse_Group_Policy_Configurations_Definition_Values": [
                                "SUCCEEDED"
                            ]
                        }
                    }
                },
                "runAfter": {
                    "Parse_Group_Policy_Configurations": [
                        "SUCCEEDED"
                    ]
                },
                "runtimeConfiguration": {
                    "concurrency": {
                        "repetitions": 1
                    }
                }
            }
        },
        "contentVersion": "1.0.0.0",
        "outputs": {},
        "triggers": {
            "When_a_HTTP_request_is_received": {
                "type": "Request",
                "kind": "Http",
                "inputs": {
                    "schema": {
                        "type": "object",
                        "properties": {
                            "tenantId": {
                                "type": "string"
                            }
                        }
                    }
                }
            }
        },
        "parameters": {
            "axiomhub_axiomcloud_name": {
                "type": "String",
                "value": "AxiomHub-AxiomCloud-APIKey",
                "defaultValue": "AxiomHub-AxiomCloud-APIKey"
            },
            "axiomhub_axiomcloud_url": {
                "type": "String",
                "value": "https://axiomhub.azurewebsites.net",
                "defaultValue": "https://axiomhub.azurewebsites.net"
            },
            "axiomhub_axiomcloud_apikey": {
                "type": "String",
                "value": "@appsetting('axiomhub_axiomcloud_apikey')",
                "defaultValue": "@appsetting('axiomhub_axiomcloud_apikey')"
            },
            "axiomitcustomerappappid": {
                "type": "String",
                "value": "@appsetting('axiomitcustomerappappid')",
                "defaultValue": "@appsetting('axiomitcustomerappappid')"
            },
            "axiomitcustomerappsecret": {
                "type": "String",
                "value": "@appsetting('axiomitcustomerappsecret')",
                "defaultValue": "@appsetting('axiomitcustomerappsecret')"
            },
            "cw_client_id": {
                "type": "String",
                "value": "@appsetting('cw_clientid')",
                "defaultValue": "@appsetting('cw_clientid')"
            },
            "cw_private_key": {
                "type": "String",
                "value": "@appsetting('cw_privatekey')",
                "defaultValue": "@appsetting('cw_privatekey')"
            },
            "cw_public_key": {
                "type": "String",
                "value": "@appsetting('cw_publickey')",
                "defaultValue": "@appsetting('cw_publickey')"
            },
            "cw_siteid": {
                "type": "String",
                "value": "@appsetting('cw_siteid')",
                "defaultValue": "@appsetting('cw_siteid')"
            },
            "cw_url": {
                "type": "String",
                "value": "https://api-aus.myconnectwise.net/v4_6_release/apis/3.0/",
                "defaultValue": "https://api-aus.myconnectwise.net/v4_6_release/apis/3.0/"
            },
            "delegatetenantId": {
                "type": "String",
                "value": "xxxxx-fb1a-4f6d-aafc-f3eb22422425",
                "defaultValue": "xxxxx-fb1a-4f6d-aafc-f3eb22422425"
            },
            "itg-apikey-pwvault": {
                "type": "String",
                "value": "@appsetting('itg-apikey-pwvault')",
                "defaultValue": "@appsetting('itg-apikey-pwvault')"
            },
            "partner_app_app_id": {
                "type": "String",
                "value": "@appsetting('PartnerAppAppID')",
                "defaultValue": "@appsetting('PartnerAppAppID')"
            },
            "partner_refresh_app_secret": {
                "type": "String",
                "value": "@appsetting('PartnerRefreshAppSecret')",
                "defaultValue": "@appsetting('PartnerRefreshAppSecret')"
            },
            "PartnerCentreApplicationPermissionsAppId": {
                "type": "String",
                "value": "@appsetting('PartnerCentreApplicationPermissionsAppId')",
                "defaultValue": "@appsetting('PartnerCentreApplicationPermissionsAppId')"
            },
            "PartnerCentreApplicationPermissionsAppSecret": {
                "type": "String",
                "value": "@appsetting('PartnerCentreApplicationPermissionsAppSecret')",
                "defaultValue": "@appsetting('PartnerCentreApplicationPermissionsAppSecret')"
            },
            "partner_refresh_token": {
                "type": "String",
                "value": "@appsetting('PartnerRefreshToken')",
                "defaultValue": "@appsetting('PartnerRefreshToken')"
            },
            "sp_aad_excluded_tenants_list_id": {
                "type": "String",
                "value": "xxxx-31d7-4e16-89ea-37979c6373d8",
                "defaultValue": "xxxx-31d7-4e16-89ea-37979c6373d8"
            },
            "sp_field_id_cw_customers_tenantid": {
                "type": "String",
                "value": "@appsetting('sp_field_id_cw_customers_tenantid')",
                "defaultValue": "@appsetting('sp_field_id_cw_customers_tenantid')"
            },
            "sp_list_id_aad_tenants": {
                "type": "String",
                "value": "@appsetting('sp_list_id_aad_tenants')",
                "defaultValue": "@appsetting('sp_list_id_aad_tenants')"
            },
            "sp_list_id_cw_customers": {
                "type": "String",
                "value": "@appsetting('sp_list_id_cw_customers')",
                "defaultValue": "@appsetting('sp_list_id_cw_customers')"
            },
            "sp_list_id_itg_customers": {
                "type": "String",
                "value": "@appsetting('sp_list_id_itg_customers')",
                "defaultValue": "@appsetting('sp_list_id_itg_customers')"
            },
            "sp_list_id_whmcs_customers": {
                "type": "String",
                "value": "@appsetting('sp_list_id_whmcs_customers')",
                "defaultValue": "@appsetting('sp_list_id_whmcs_customers')"
            },
            "sp_list_id_whmcs_nce_overrides": {
                "type": "String",
                "value": "@appsetting('sp_list_id_whmcs_nce_overrides')",
                "defaultValue": "@appsetting('sp_list_id_whmcs_nce_overrides')"
            },
            "sp_list_id_whmcs_products": {
                "type": "String",
                "value": "@appsetting('sp_list_id_whmcs_products')",
                "defaultValue": "@appsetting('sp_list_id_whmcs_products')"
            },
            "sp_list_url": {
                "type": "String",
                "value": "@appsetting('sp_list_url')",
                "defaultValue": "@appsetting('sp_list_url')"
            },
            "whmcs_identifier": {
                "type": "String",
                "value": "@appsetting('whmcs_identifier')",
                "defaultValue": "@appsetting('whmcs_identifier')"
            },
            "whmcs_secret": {
                "type": "String",
                "value": "@appsetting('whmcs_secret')",
                "defaultValue": "@appsetting('whmcs_secret')"
            },
            "whmcs_url": {
                "type": "String",
                "value": "@appsetting('whmcs_url')",
                "defaultValue": "@appsetting('whmcs_url')"
            },
            "Test": {
                "type": "String",
                "value": "AAD_x0020_Tenant_x003a_Tenant_x0",
                "defaultValue": "AAD_x0020_Tenant_x003a_Tenant_x0"
            }
        }
    },
    "kind": "Stateful"
}
```

### Browser

Logic Apps (Standard) Extension 2.58.0

VSCode
Version: 1.81.1
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:40:25.698Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.6.0

AB#24945557

**Comment by Eric-B-Wu** [comment]
@axiomitdavid thanks for reporting this issue, seems like the issue is that we hotfixed serialization settings in 2.55.7 and had planned it to be in our release of 2.59.0, but the releases in between were not hotfixed. I'll work with @ccastrotrejo to either update vscode release to be 2.59.0 or hotfix it back to 2.55.7

**Comment by ccastrotrejo** [comment]
Hi @axiomitdavid , we have released a hotfix for the 2.58 version. Please update it. Thanks!
